### ISSUE TITLE
Serve the brotli compressed JS assets

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#9761]
**Is your feature request related to a problem? Please describe.**
With webpack, we compress js assets using gzip and brotli however we don't serve the the brotli assets where possible.

**Describe the solution you'd like**
We should look at these 2 possible solution

- enable brotli in akamai
- use a service worker to enable brotli

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by eagerterrier** [comment]
I've just done a little experiment and it's not _great_ news...

I uploaded a dummy css file compressed with brotli to S3 (allowed public access to this file for reasons you'll see in a sec)

https://news-assets-publicbucket-8q5knurn0a7g.s3.eu-west-1.amazonaws.com/include/vjassets/css/vendor/riddle-1.0.0.br.css

This has correct `Content-Encoding` of `br`

https://news.files.bbci.co.uk/include/vjassets/css/vendor/riddle-1.0.0.br.css <- akamai is un-brotli-ing this asset

I've compared to https://static.files.bbci.co.uk/ws/simorgh-assets/public/static/js/modern.shared-gqrj5Fb%2BGJCWU1vRjtAbN45abxc%3D.13f3b099.js.br which is registering as `content-type: binary/octet-stream` so it's nothing to do with Akamai trying to see if the object is a text type and then un-brotli-ing. This seems to be a blanket un-brotli.

So, seems like we need _something_ done at Akamai's end. This isn't option 1 (get Akamai to turn on automatic brotli compression), but rather option 2 seems to rely on Akamai flicking a switch somehow.
